js/modules/tamagotchi.js

```diff
@@ -1,16 +1,17 @@
 let energyDecreaseRate = 0.5;
-const actionBtns = document.querySelector('.action-btns')
-const restartBtn = document.querySelector('.restart')
+const actionBtns = document.querySelector('.action-btns');
+const restartBtn = document.querySelector('.restart');
+
 export default class Tamagotchi {
 	constructor() {
 		this.state = 'happy';
-		this.health = { value: 10, importance: 1 };
-		this.hunger = { value: 10, importance: 3 };
+		this.health = { value: 1, importance: 1 };
```

Comment:
**issue:** I think you have it for testing purpose, but it should be 10 as default